projects/storefrontapp-e2e-cypress/cypress/support/cx-config.commands.ts

```diff
@@ -0,0 +1,21 @@
+import { StorefrontConfig } from '@spartacus/storefront';
+
+declare namespace Cypress {
```

Comment:
Commands that have `imports` needs a little bit different `TS` declaration to work correctly. Take a look at login command.